tests/runtime/_resources.py

```
@@ -619,6 +621,59 @@ def cancel_job(self, job_qrn: str) -> None:
         """Cancels a specific quantum job."""
         # Mock implementation - no-op for testing
 
+    # Batch methods
+    _batch_counter: int = 0
+
+    def __init__(self):
```

Ruff (0.15.7)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)